geom_test.go

```diff
@@ -453,3 +465,49 @@ func TestUserData(t *testing.T) {
 	assert.Equal(t, g, g.SetUserData(1))
 	assert.Equal(t, uintptr(1), g.UserData())
 }
+
+func TestMakeValid(t *testing.T) {
+	for _, tc := range []struct {
+		name              string
+		wkt               string
+		expectedValidWkt1 string
+		expectedValidWkt2 string
+		expectedValidWkt3 string
+		expectedValidWkt4 string
```

Comment:
What do the 1, 2, 3, and 4 suffixes to `expectedValidWkt` mean here? Please give them more meaningful names.

Reply:
I gave a little more meaningful names trying to keep them short :-)